Gemfile

```diff
@@ -20,11 +22,11 @@ group :development, :test do
   gem 'pry'
   gem 'pry-nav'
   gem 'rspec-rails', '2.99'
+  gem 'shoulda-matchers', require: false
 end
 
 group :test do
   gem 'database_cleaner', '1.2.0'
-  gem 'shoulda-matchers', require: false
```

Comment:
I'm not sure why you moved these from test to development. They belong in the test group, as shown in the docs.